src/components/PoolSelect/PoolSelect.tsx

```diff
@@ -37,14 +39,16 @@ export const PoolSelect: React.FC<Props> = ({
   onChangeValue,
   inputProps = {},
 }) => {
+  const [isInitialized, setIsInitialized] = useState<boolean>(false);
   const [searchText, setSearchText] = useState<string>('');
   const [isDropdownOpen, setIsDropdownOpen] = useState<boolean>(false);
 
   useEffect(() => {
-    if (value !== undefined && searchText === '') {
+    if (value !== undefined && searchText === '' && !isInitialized) {
       setSearchText(getPoolAssetsRepr(value));
+      setIsInitialized(true);
     }
-  }, [value, searchText]);
+  }, [value, searchText, isInitialized]);
```

Comment:
What `isInitialized` states for?

Reply:
it will allow user delete value from input. 
if we will delete isInitialized state, react will set value to the input every time when it is empty.

```
if (value !== undefined && searchText === '') {
   setSearchText(getPoolAssetsRepr(value));
}
```